### ISSUE TITLE
Tests running slowly after just installing a couple icons

### ISSUE THREAD

**Issue by mateusjunges** [blade-ui-kit/blade-icons#234]
- Library version: 1.6.0
- Laravel version: 11.6
- PHP version: 8.3.4

TLDR: After installing 4 blade icons packages, the test suite in a fresh laravel app with breeze installed takes 40 seconds to run, even using parallel testing.

### Description:
2 days ago I was helping someone that was having issues to setup tests on a new project. It was a small to medium size Laravel 11 app, a lot of code written but the only tests there was the tests that are published when installing breeze. It took 40 sec to run a few simple breeze auth tests (using parallel testing):
```
.........................                                         25 / 25 (100%)

Time: 00:39.959, Memory: 28.00 MB

OK (25 tests, 61 assertions)
```

Since there was no tests other than breeze, I created a fresh laravel 11 app, with breeze (blade & alpine) and tried running the tests there. It took `00:00.611` to run the same tests.

I thought one of the dependencies were slowing things down, and started doing composer require on each dependency separately, and removing it when it did not make any difference in the tests run time.
After a couple composer require/composer remove, I found that some blade-ui-kit dependencies were causing the issue. This is what I ended up with in composer.json other than the default in a laravel  11 fresh installation:
```
"require": {
    "codeat3/blade-clarity-icons": "^1.9",
    "codeat3/blade-phosphor-icons": "^2.2",
    "owenvoke/blade-fontawesome": "^2.6",
    "ryangjchandler/blade-tabler-icons": "^2.3"
}
```

After source diving into `blace-icons` code, I found that 
https://github.com/blade-ui-kit/blade-icons/blob/9d3fbda75b935dc58e22ccdbb816f6cba91bf94c/src/Factory.php#L99-L116
 and https://github.com/blade-ui-kit/blade-icons/blob/9d3fbda75b935dc58e22ccdbb816f6cba91bf94c/src/BladeIconsServiceProvider.php#L65-L67

is causing the issue. 

`codeat3/blade-phosphor-icons` has +8k icons, which makes it really slow. 

### Steps to reproduce

- laravel new test-app
- composer require laravel/breeze --dev
- Install breeze with blade & alpine
- Install the 4 packages I posted above on the json snipped
- Run the tests and see the issue

**Comment by driesvints** [comment]
Did you run this with caching? https://github.com/blade-ui-kit/blade-icons?tab=readme-ov-file#caching

**Comment by mateusjunges** [comment]
Hmm I did not. Just tried that and the tests run normally after caching. This is not my codebase, and I don't think the person who added the icons have run the cache command, they just searched for an icon and run the `composer require` command on the icon page (e.g https://blade-ui-kit.com/blade-icons/fileicon-pypi). Maybe it is worth it mentioning this on the icon page?

Anyway, running the cache command fixed it. Thank you Dries!
